bindgen/ir/types/PointerType.cpp

```diff
@@ -23,3 +23,14 @@ bool PointerType::operator==(const Type &other) const {
     }
     return false;
 }
+
+bool PointerType::usesOpaqueType() const { return type->usesOpaqueType(); }
```

Comment:
A pointer to an opaque type is always sound, no?

Reply:
Yes, but by the time this method is called, all pointers to opaque type are replaced with pointers to `Byte`, so this method is used to finds illegal usages of opaque type